charts/pega/README.md

```diff
@@ -658,6 +658,21 @@ If you are planning to use Cassandra (usually as a part of Pega Customer Decisio
 
 To use an existing Cassandra deployment, set `cassandra.enabled` to `false` and configure the `dds` section to reference your deployment.
 
+The following parameters are available for configuration of the connection to the external Cassandra cluster.
+
+Parameter     | Description                                                                                                                                                                                                                                                                                                                   | Default value
+---           |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ---
+`externalNodes`    | A comma separated list of hosts in the Cassandra cluster. | Empty
+`port` | TCP Port to connect to cassandra. | 9042
+`username` | The username for authentication with the Cassandra cluster. | dnode_ext
```

Comment:
not sure what this means here and in the password parameter>> dnode_ext

Reply:
dnode_ext is the default value used for the username and password to authenticate to a cassandra cluster. I'm not sure that it makes sense to change this one.

Reply:
I'm asking you to clarify how dnode_ext translates in a text string for use in these parameters that require a text string, not get rid of the value if that's what the file requires. but humans are not gonna understand how these various values that require text strings to see how it's resolved by dnode_ext...which kinda looks like an endpoint on a server. Just saying...